storage/tianmu/handler/tianmu_handler_com.cpp

```diff
@@ -211,10 +211,10 @@ int rcbase_init_func(void *p) {
   ha_rcengine_ = nullptr;
 
   try {
-    std::string log_file = mysql_home_ptr;
-    log_setup(log_file + "/log/tianmu.log");
-    rc_control_.addOutput(new system::FileOut(log_file + "/log/trace.log"));
-    rc_querylog_.addOutput(new system::FileOut(log_file + "/log/query.log"));
+    std::string log_file = mysql_real_data_home_ptr;
+    log_setup(log_file + "/tianmu_engine.log");
```

Comment:
keeping the name as the old one.  It's not necessary to be with `engine` suffix.

Reply:
Fixed, please review again.